public/base-style.css

```diff
@@ -88,7 +88,7 @@ li { padding-top: .3rem; padding-bottom: .3rem; }
 
 pre, code, samp { font-family: "Source Code Pro", Consolas, monospace; font-size: inherit; }
 
-pre { margin-top: 0; margin-bottom: 1rem; overflow-x: scroll; max-width: calc(100vw - calc(var(--containerPadding) * 2))}
+pre { margin-top: 0; margin-bottom: 1rem; overflow-x: scroll; max-width: var(--containerWidth);}
```

Comment:
One more nitpick (sorry for not thinking of this in the previous comment) - this `base-style.css` file is more about colors and fonts and paddings for things, where the `layout.css` file is about the width and alignment of things.

So, I think this style

```css
pre {
	max-width: var(--containerWidth);
}
```
should go in the `layout.css` file instead of here in the `base-style.css` file.